addon/initializers/setup-session.js

```diff
@@ -1,7 +1,9 @@
 import Session from '../session';
 import Configuration from '../configuration';
+import TestAuthenticator from '../authenticators/test';
 
 export default function setupSession(registry) {
   registry.register('simple-auth-session:main', Session);
+  registry.register('simple-auth-authenticator:test', TestAuthenticator);
```

Comment:
The Test authenticator should only be registered in test env. Is it actually possible to have the authenticator in `test-support` that would then only be loaded in test mode, e.g. `test-support/authenticators/test.js`?